my_game.py

```diff
@@ -272,24 +259,6 @@ def on_key_press(self, key, modifiers):
             self.window.show_view(game_view)
             print("Game Reset! 🔁 -- Turn Debug mode off to remove this feature! ✔")
 
-    def on_key_release(self, key, modifiers):
```

Comment:
This function should *NOT* be deleted. It's what's causing the problems you are having.